src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -457,7 +340,6 @@ of `pan-x | pan-y | manipulation` you prevent the default double-tap
 behavior.
```

Comment:
This "|" syntax is possibly confusing.  Maybe just say "manipulation" (since that's the one that disables double-tap without impacting anything else)?  `manipulation` is also [supported by](https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away?hl=en) Chrome, Safari and Firefox now too.